<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>651</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>651</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>551</width>
    <height>651</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Detalle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>531</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Registro:</string>
    </property>
    <widget class="QTableWidget" name="tblDetalle">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>501</width>
       <height>331</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>11111</width>
       <height>50000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID_Detalle</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cuota</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Concepto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha Detalle</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Monto Detalle</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Observaciones</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Datos:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_Detalle:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIdDetalle">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cuota:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Concepto:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha Detalle:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Monto Detalle:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMonto">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Observaciones</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtObservacion">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="cboCuota">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cboConcepto">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txFeDetalle">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Botones:</string>
    </property>
    <widget class="QPushButton" name="btnRegistrar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnActualizar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLimpiar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
      </font>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
